src/App.php

```diff
@@ -26,6 +26,7 @@
 use Psr\Http\Message\ResponseInterface;
```

Comment:
@abbadon1334 please explain this coverage change:

![image](https://github.com/atk4/ui/assets/2228672/5f3fe01e-f422-4887-a3f7-c4337ecdcb34)

- it seems some error is newly triggered - how to repro it?
- it should NOT be triggered by these refactoring changes
- a Behat test to actually trigger it will be good, as a separate PR please